plugins/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "appium-inspector-plugin",
+  "version": "2024.12.1",
+  "description": "An app inspector for use with an Appium server",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/appium/appium-inspector.git"
+  },
+  "author": {
+    "name": "Appium Developers",
+    "url": "https://github.com/appium"
+  },
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/appium/appium-inspector/issues"
+  },
+  "keywords": [
+    "appium"
+  ],
+  "homepage": "https://github.com/appium/appium-inspector",
+  "main": "index.js",
```

Comment:
ESM modules require `exports` property to be defined. The `main` one is relevant for CJS